commands/release.js

```diff
@@ -60,8 +60,8 @@ export async function command() {
 	}
 
 	const stableVersions = versions.filter(version => {
-		const v = coerce(version);
-		return v.prerelease.length === 0;
+		const prereleaseComponents = prerelease(version);
+		return prereleaseComponents.length === 0;
```

Comment:
I don't believe this will work as semver.prerelease returns `null` if the version has no prerelease identifier.

```
> semver.prerelease('v1.0.0')
null

> semver.prerelease('v1.0.0').length
Uncaught TypeError: Cannot read property 'length' of null
```


We could check if the value is null and that indicates it is a stable version and not a prerelease.

```suggestion
		return prerelease(version) === null;
```

Reply:
yikes, thorough reviewing, good catch, thanks